Sources/Query.swift

```diff
@@ -169,7 +517,175 @@ public struct Query<ContentType: ContentModel> {
     }
 }
 
-extension String {
+/// Queries on Asset types. All methods from Query, and therefore ChainableQuery, are inherited and available.
+public final class AssetQuery: Query {
```

Comment:
Does `final` in this case mean, a user cannot create her own query, aka extend this Query? If so why would we like to limit her?